packages/xsnap/src/xsnap.c

```diff
@@ -84,7 +86,8 @@ static char* fxWriteNetStringError(int code);
 #define mxSnapshotCallbackCount 6
 txCallback gxSnapshotCallbacks[mxSnapshotCallbackCount] = {
 	fx_issueCommand, // 0
-	fx_Array_prototype_meter, // 1
+	// fx_Array_prototype_meter, // 1
+	NULL, // 1
```

Comment:
Does this mean that we cannot load any snapshots that happened to include a reference to this function? Or, we can load them, but if they subsequently actually try to call it, we'll get a crash?

I'm wondering if it would be better to remove the callback slot entirely, rather than replacing it with a NULL, if we aren't really preserving snapshot compatibility across this change.

Reply:
Right; the NULL isn't worth the risk.

fixed in f337f8a